cmd/serf/command/agent/ipc.go

```diff
@@ -605,6 +606,17 @@ func (i *AgentIPC) handleForceLeave(client *IPCClient, seq uint64) error {
 	}
 
 	// Attempt leave
+	if req.Prune {
+		err := i.agent.ForceLeavePrune(req.Node)
+
+		// Respond
+		resp := responseHeader{
+			Seq:   seq,
+			Error: errToString(err),
+		}
+		return client.Send(&resp, nil)
+	}
```

Comment:
Probably for clarity the remainder of this method should be in an `} else {` block since this flag implies an either/or execution.